### DISCUSSION TITLE
Imposing second derivative boundary condition

### DISCUSSION THREAD

**Discussion by phdstudentjw** [lululxvi/deepxde#852] [General]
The documentation explains very clearly how to impose various boundary conditions. However, suppose I'm solving a one-dimensional ODE `u(x)` and I want to impose the second derivative of `d2u(x)/dx2 = 3`. How can I do that? Is this possible through deepxde?

Thank you.

**Comment by lululxvi** [comment]
https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/ode.2nd.html
